Making event-bus examples CE spec 3.0 compliant (#129)
kyma-project/examples: event-subscription/service/README.md

    ```
    > **NOTE:** To send multiple Events, change the `orderCode` to have a unique value for each POST request.

> 10. To access attributes of events which are compliant to CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md), you can access event object as follows:

Reviewer: ```suggestion
10. To access Event attributes compliant with the CloudEvents [specification](https://github.com/cloudevents/spec/blob/v0.3/http-transport-binding.md) use the following paths in the request:
```
